### PR TITLE
Fork new feedstock

### PR DESCRIPTION
**Pull request by Arishamays1**
Added `pkgutil-resolve-name` to AnacondaRecipes from Conda-forge.
- removed noarch
- updated about section

### CODE REVIEW

**Comment by pseudoyim** [comment]
The version pinnings on `python` will need to be removed here: https://github.com/AnacondaRecipes/pkgutil-resolve-name-feedstock/pull/1/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR21-R23

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ about:
   home: https://github.com/graingert/pkgutil-resolve-name
   summary: Backport of Python 3.9's pkgutil.resolve_name; resolves a name to an object.
   license: BSD-4-Clause
-  license_file: LICENSE
```
Comment (cbouss) [inline recipe/meta.yaml:37]: license should be MIT AND PSF-2.0
license_family should be PSF
license_file should be LICENSE (to match https://github.com/graingert/pkgutil-resolve-name/blob/default/LICENSE)
(conda build looks for the license file first in the downloaded sources, then in the feedstock)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ about:
...
+  license_file: LICENSE.txt
```
Comment (cbouss) [inline recipe/meta.yaml]: Python pinning should be removed.
```
requirements:
  host:
    - flit-core
    - pip
    - python
  run:
    - python
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ about:
   home: https://github.com/graingert/pkgutil-resolve-name
```
Comment (cbouss) [inline recipe/meta.yaml:36]: Since this is a backport of a python module, there should be a skip for python higher than 3.8.

**Review by cbouss — DISMISSED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:37]: Got it! I'll make that amendment now.

**Comment by Arishamays1** [comment]
> The version pinnings on `python` will need to be removed here: https://github.com/AnacondaRecipes/pkgutil-resolve-name-feedstock/pull/1/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR21-R23

Thanks Paul! I'll make this change now.

**Review by cbouss — APPROVED** [review]
LGTM, thank you!

**Pull request merged**